library/unistd/spawnvpe.c

```diff
@@ -288,6 +293,7 @@ spawnvpe(
     D(("(*)Calling SystemTags.\n"));
```

Comment:
Maybe this line needs to get after `#if USE_CNPT` otherwise it will be confusing for users who check the debug

Reply:
So, we might need one for `CreateNewProcTags` after line 297 and this one moved after line 336